spec.bs

```diff
@@ -765,13 +765,16 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
 1. Let |bidIgs| be a new [=list=] of [=interest groups=].
 1. If |config|["{{AuctionAdConfig/signal}}"] [=map/exists=], then:
   1. Let |signal| be |config|["{{AuctionAdConfig/signal}}"].
-  1. If |signal| is [=AbortSignal/aborted=], then [=reject=] |p| with |signal|'s
-    [=AbortSignal/abort reason=] and return |p|.

```

Comment:
Does changing this part do anything, or is just for uniformity?

Reply:
Yes. There are a couple of reasons for this change:
1. It ensures that interest group update/bid counts occur in parallel instead of the main JS thread (which is where the Abort handler executes).
2. It makes it so we update interest groups/bid counts if the abort happens before runAdAuction is called. 
Both of these changes provide a more consistent experience for developers and better match the implementation in Chrome.

Reply:
I don't think (2) is true at all? We just exit if it's already aborted, and don't do anything:

https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/ad_auction/navigator_auction.cc;drc=7578f90a7173017ec95ded2bb21419ab14e57a16;l=3787